pkgs/development/tools/replay-io/default.nix

```diff
@@ -0,0 +1,144 @@
+{ stdenv, lib, fetchurl, fetchgit, autoPatchelfHook, makeWrapper, libcxx, libX11
+, libXt, libXdamage, glib, gtk3, dbus-glib, openssl, nodejs, zlib }:
+let metadata = lib.importJSON ./meta.json;
+in rec {
+  replay-recordreplay = stdenv.mkDerivation rec {
+    pname = "replay-recordreplay";
+    version = builtins.head (builtins.match ".*/linux-recordreplay-(.*).tgz"
+      metadata.recordreplay.url);
+    nativeBuildInputs = [ autoPatchelfHook ];
+    buildInputs = [ stdenv.cc.cc.lib openssl zlib ];
+
+    src = (fetchurl metadata.recordreplay);
+    unpackPhase = ''
+      tar xzf $src
+    '';
```

Comment:
Isn't that done automatically?

Reply:
The tarball does not contain a subdirectory so without this I would be getting `unpacker appears to have produced no directories`.

Reply:
You want to use stripRoot instead IIRC.

Reply:
Didn't know about that one, thanks for the hint - added it!